server/views/index.jade

```diff
@@ -16,10 +16,7 @@ block content
             p.help= t('index noscript message')
 
 block js
-    script(src="/scripts/vendor.js")
     if env
         script.
-            require.register('env', function(exports, require, module) {
-              module.exports = !{JSON.stringify(env)}
-            })
-    script(src="/scripts/app.js", onload="require('initialize');")
+            ENV = !{JSON.stringify(env)};
+    script(src="/app.js" defer=true)
```

Comment:
Webpack offers a way to append a hash to generated files, but I ask myself if using a cache-manifest (using https://github.com/lettertwo/appcache-webpack-plugin) wo'nt be a better way to do this…